docs/source/CVAOpt_Algorithm/Parameters_and_Hyperparameters.md

```diff
@@ -42,3 +42,50 @@ The number of parents in a iteration
 
 The number of offsprings created in a iteration
 
+## Example
+```python
+from tq42.client import TQ42Client
+from tq42.experiment_run import ExperimentRun
+from tq42.algorithm import AlgorithmProto
+from tq42.compute import HardwareProto
+
+client = TQ42Client()
+client.login()
+
+# set your function evaluation
+func_eval_worker_ip = '34.32.169.11'
+# single-objective optimization
+func_eval_worker_url_sphere = 'http://' + func_eval_worker_ip + ':8000/test_func_eval/Sphere'
+# multi-objective optimization
+func_eval_worker_url_zdt1 = 'http://' + func_eval_worker_ip + ':8000/test_func_eval/ZDT1'
+
+# set your cva params
+cva_params = {}
+cva_params['objectives'] = [{'name': 'Sphere', 'aim_type':'MINIMIZE'}] 
+# set optimisation vars
+cva_params['variables'] = []
+cva_params['variables'].append({'name': 'x1', 'info_real':{'lower_bound':-1.0, 'upper_bound':1.0}})
+cva_params['variables'].append({'name': 'x2', 'info_real':{'lower_bound':-1.0, 'upper_bound':1.0}})
+# set URL for evaluation function
+cva_params['func_eval_worker_url'] = func_eval_worker_url_sphere
+# set cva parameters
+cva_params['parameters'] = {}
+# set the number of generations (iterations)
+cva_params['parameters']['max_generation'] = 250
+# set the ES specific parameters mue and lambda
+cva_params['parameters']['mue'] = 15
+cva_params['parameters']['lambda'] = 100
+
+# run experiment
+ExperimentRun.create(
+    client=client, 
+    algorithm=AlgorithmProto.CVA_OPT, 
+    experiment_id=experiment_id,
+    compute=HardwareProto.SMALL, 
+    parameters={'parameters': cva_params, 'inputs': {} }
+)
+```
+
+```python
```

Comment:
why is there an empty python code block here?

Reply:
removed it